### PR TITLE
Limit modules test with Jenkins and Profile change

### PR DESCRIPTION
**Pull request by SarikaSinha**
## Author checklist

- [x] I have thoroughly tested my changes
- [x] The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- [x] I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)

### CODE REVIEW

**Comment by SarikaSinha** [comment]
Fixes https://github.com/eclipse-jdt/eclipse.jdt.debug/issues/65

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ pipeline {
 				wrap([$class: 'Xvnc', useXauthority: true]) {
 					sh """
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
-						-Pbuild-individual-bundles -Pbree-libs -Papi-check\
+						-Pbuild-individual-bundles Ptest-on-javase-19 -Pbree-libs -Papi-check\
```
Comment (sravanlakkimsetti) [inline Jenkinsfile:21]: should be `-Ptest-on-javase-19` - is missing

**Review by sravanlakkimsetti — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (SarikaSinha) [reply Jenkinsfile]: Thanks

**Review by sravanlakkimsetti — APPROVED** [review]

**Pull request merged**